modules/friendlytag.js

```diff
@@ -545,14 +563,14 @@ Twinkle.tag.updateSortOrder = function(e) {
 		});
 	};
 
+
 	if (sortorder === 'cat') { // categorical sort order
 		// function to iterate through the tags and create a checkbox for each one
-		var doCategoryCheckboxes = function(subdiv, array) {
+		var doCategoryCheckboxes = function(subdiv, subgroup) {
```

Comment:
A little annoying: the `Array.isArray` will pass `group` to this while the `else` will pass `subgroup`.  I think it's worth it, but, still, slightly confusing.  I should add a comment to that effect.